exchanges/coinbasepro/coinbasepro_types.go

```diff
@@ -125,7 +125,7 @@ type GeneralizedOrderResponse struct {
 	Funds          float64   `json:"funds,string"`
 	SpecifiedFunds float64   `json:"specified_funds,string"`
 	DoneReason     string    `json:"done_reason"`
-	DoneAt         string    `json:"done_at"`
+	DoneAt         time.Time `json:"done_at"`
```

Comment:
Reference: https://docs.cloud.coinbase.com/exchange/reference/exchangerestapi_getorders

CreatedAt and DoneAt has same date-time type